src/Makefile.am

```diff
@@ -262,6 +262,7 @@ libbitcoin_server_a_SOURCES = \
   torcontrol.cpp \
   txdb.cpp \
   txmempool.cpp \
+  upgrade.cpp \
```

Comment:
Also need to add `upgrade.h` to `BITCOIN_CORE_H`.

Reply:
Added above